cpp/src/slice2js/Gen.cpp

```diff
@@ -1552,15 +1550,13 @@ Slice::Gen::TypesVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
             //  0: servant method name in case of a keyword conflict (e.g., "_while"),
             //     otherwise an empty string
             //  1: mode (undefined == Normal or int)
-            //  2: sendMode (undefined == Normal or int)
-            //  3: amd (undefined or 1)
```

Comment:
Looking at the implementation, I didn't see anywhere where we send `amd`,
so I believe this comment was just outdated, and removed it while I was in here.

It's worth noting that in the other wrapper languages I looked at, we do send `amd`... just not JS.